astra-bulk-edit.php

```diff
@@ -19,7 +19,7 @@
 /**
  * Set constants.
  */
-define( 'ASTRA_BLK_VER', '1.2.9' );
+define( 'ASTRA_BLK_VER', '1.3.0' );
```

Comment:
```suggestion
define( 'ASTRA_BLK_VER', '1.2.10' );
```

Reply:
As this is a minor fix, let's not update the major version. Be it with v1.2.10

Reply:
Thank you. I have updated the versions.